src/ORM/Table.php

```diff
@@ -1048,6 +1048,27 @@ protected function _setFieldMatchers($options, $keys)
 
         return $options;
     }
+    
+    /**
+     * get value of a single column
+     * 
+     * @param string|array $field string field or key value pair array
+     * @param array $conditions
+     * @param array $order
+     * @return mixed|null null field value or null is returned
+     */
+    public function field($field, $conditions = [], $order = [])
+    {
+        $result = $this->find()->select(is_array($field) ? $field : [$field])->where($conditions)->order($order)->first();
```

Comment:
You can also just cast instead of `is_array($field) ? $field : [$field]`:

     (array)$field